<?xml version="1.0" ?>

<launch>
<group ns="hog">
  <!-- params -->
  <param name="forest_file" value="$(find leg_tracker)/config/trained_leg_detector_res=0.33.yaml" />
  <param name="scan_topic" value="/hog/scan0" />
  <param name="fixed_frame" value="/map"/>
  <param name="publish_people_frame" value="/map"/>
  <param name="scan_frequency" value="10"/>
  <param name="display_detected_people" value="true"/>
  <param name="use_scan_header_stamp_for_tfs" value="true"/>
  <param name="dist_travelled_together_to_initiate_leg_pair" value=".1"/>

  <!-- run detect_leg_clusters -->
  <node pkg="leg_tracker" type="detect_leg_clusters" name="detect_leg_clusters" output="screen"/>

  <!-- run joint_leg_tracker -->
  <node pkg="leg_tracker" type="joint_leg_tracker.py" name="joint_leg_tracker" output="screen"/>

   <!-- run local_occupancy_grid_mapping -->
  <node pkg="leg_tracker" type="local_occupancy_grid_mapping" name="local_occupancy_grid_mapping" output="screen"/>
</group>
<group ns="mouse">
  <!-- params -->
  <param name="forest_file" value="$(find leg_tracker)/config/trained_leg_detector_res=0.33.yaml" />
  <param name="scan_topic" value="/mouse/scan0" />
  <param name="fixed_frame" value="/map"/>
  <param name="scan_frequency" value="10"/>
  <param name="display_detected_people" value="true"/>
  <param name="use_scan_header_stamp_for_tfs" value="true"/>
  <param name="dist_travelled_together_to_initiate_leg_pair" value=".1"/>


  <!-- run detect_leg_clusters -->
  <node pkg="leg_tracker" type="detect_leg_clusters" name="detect_leg_clusters" output="screen"/>

  <!-- run joint_leg_tracker -->
  <node pkg="leg_tracker" type="joint_leg_tracker.py" name="joint_leg_tracker" output="screen"/>

   <!-- run local_occupancy_grid_mapping -->
  <node pkg="leg_tracker" type="local_occupancy_grid_mapping" name="local_occupancy_grid_mapping" output="screen"/>
</group>
<group ns="snake">
  <!-- params -->
  <param name="forest_file" value="$(find leg_tracker)/config/trained_leg_detector_res=0.33.yaml" />
  <param name="scan_topic" value="/snake/scan0" />
  <param name="fixed_frame" value="/map"/>
  <param name="scan_frequency" value="10"/>
  <param name="display_detected_people" value="true"/>
  <param name="use_scan_header_stamp_for_tfs" value="true"/>
  <param name="dist_travelled_together_to_initiate_leg_pair" value=".1"/>


  <!-- run detect_leg_clusters -->
  <node pkg="leg_tracker" type="detect_leg_clusters" name="detect_leg_clusters" output="screen"/>

  <!-- run joint_leg_tracker -->
  <node pkg="leg_tracker" type="joint_leg_tracker.py" name="joint_leg_tracker" output="screen"/>

   <!-- run local_occupancy_grid_mapping -->
  <node pkg="leg_tracker" type="local_occupancy_grid_mapping" name="local_occupancy_grid_mapping" output="screen"/>
</group>
</launch>
